spring-threads/README.md

```diff
@@ -0,0 +1,10 @@
+## Spring Threads
+
+This module contains articles about threading using threads, specifically Spring's ThreadPoolTaskExecutor
+
+### Relevant Articles:
+- [Introduction to Thread Pools in Java](https://www.baeldung.com/thread-pool-java-and-guava)
+- [How To Do @Async in Spring](https://www.baeldung.com/spring-async)
+- [A Guide to the Spring Task Scheduler](https://www.baeldung.com/spring-task-scheduler)
+- [A Guide to the Java ExecutorService](https://www.baeldung.com/java-executor-service-tutorial)
+
```

Comment:
The module should only contain links to articles that link to this module. Since this is a new module, it should only have a link to your article. The link to your article will be https://baeldung.com/threadpool-task-executor-corepoolsize-maxpoolsize